### DISCUSSION TITLE
自己编译参考文献和研究成果的问题

### DISCUSSION THREAD

**Discussion by lujingping** [YanMing-lxb/GUET_Thesis_LaTeX#36] [提问区] [answered]
您好！请问本地编译参考文献和研究成果怎么处理？原先小论文我已经写成Latex文件了，所以想直接把一部分内容复制粘贴过来大论文，参考文献不通过生成的方式，我直接按章节那样放进去，但标题不出现参考文献，而出现“Bibliography”。盼回复，非常感谢！

**Comment by YanMing-lxb** [comment]
这个学位论文模板的参考文献是用的biblatex，期刊论文投稿一般用的是bibtex，所以你检查一下你的bib文件是否是biblatex所用的格式

> 参考文献不通过生成的方式，我直接按章节那样放进去
这个是指，你没有采用biblatex或者bibtex来处理文献，而是第三种方案手动管理的是嘛？

**Reply by lujingping** [reply]
是的，我是手动管理的。我还不知biblatex或者bibtex怎么用，能否提供手动管理的版本？

**Reply by YanMing-lxb** [reply]
😂，这个，我觉得你还是稍微学一下比较好，都是现成的，手动管理确实比较麻烦，而且参考文献格式需要你自己弄。
能发给我一个样例吗？我看看手动管理是怎么样的，手动管理我也没学习过，看怎么回事

**Reply by lujingping** [reply]
\begin{thebibliography}{99}
\bibitem{Algaba2015} Algaba, A., Dominguez-Moreno, M.C., Merino, M., Rodriguez-Luis, A.J.: Study of the Hopf Bifurcation in the Lorenz,
Chen  and  L{\"u} systems. Nonlinear Dynam. {\textbf {79}},  885-902 (2015)

\bibitem{Bautin1952} Bautin, N.: On the number of limit cycles appearing from an equilibrium point of the focus or center type under varying coefficients. Matem. Sb. {\textbf {30}},  181-196 (1952)

\bibitem{Barreira}Barreira, L.,  Llibre, J.,  Valls, C.:  {Bifurcation of
limit cycles from a 4-dimensional center in ${\mathbb R}^m$ in
resonance $1: N$}.  {J. Math. Anal. Appl.} {\textbf  {389}},
 754-768 (2012)
\end{thebibliography}
我原先用tex写的小论文，都是在文章里直接输入参考文献，不同刊物参考文献的格式不一样，所以我就直接在论文里编辑和改动。现在大论文有100多篇参考文献，如果用biblatex或者bibtex是不是还需要重新编辑一遍？本来我打算直接复制粘贴进大论文里。麻烦您帮看看，非常感谢！

**Reply by YanMing-lxb** [reply]
你之前是不是对于不同刊物的参考文献格式要求，手动去修改bibitem内容的格式啊

**Reply by YanMing-lxb** [reply]
如果是这样的话，我建议你还是直接转成biblatex吧……要不然，你这工作量老大了

**Reply by lujingping** [reply]
是的，我用的最笨的方法，那个biblatex或者bibtex怎么用，哪个软件编辑？可否提供一下，我学一下怎么使用，谢谢！

**Reply by YanMing-lxb** [reply, marked as answer]
这个你去b站简单学一下 Zotero 和 latex 管理参考文献的两种方式：biblatex、bibtex（西北农林大学老师弄得那个教程）。

我是用`zotero`来管理文献，然后就可以直接通过`Better BibTeX for Zotero`导出所需要的bib文件。至于在这个模板里的使用，我的示例和注释里都有写明，仔细看一下模板即可。
文献的话，我一般是网上找到了，用`zotero`的网页插件直接抓取到，或者用文献的doi直接获取。

你现在的这种情况的话，我推荐是先把你的小论文放到`zotero`里，然后用`Zotero Reference`这个插件来抓取你所引用的论文信息。最后如果有缺的再手动补充，最后导出`biblatex`格式的bib文件即可。至于key的问题就没办法了，可能需要你自己手动调整

zotero 插件的话，我用的插件列表如下，你可以都装：
![image](https://github.com/user-attachments/assets/d8140d3c-e680-4c9b-801d-fd1b1bdd4b17)

优先装插件市场： [Add-on Market for Zotero](https://github.com/syt2/zotero-addons)
其他的可以通过插件市场装。

**Reply by lujingping** [reply]
好的，非常感谢！

**Reply by lujingping** [reply]
您好！我昨晚安装了Zotero 7，并学习了使用，但是在抓取我小论文文献时，参考文献出不来，下载了好多个zotero reference插件安装，但是参考文献出不来，今天我卸载了Zotero 7, 安装了Zotero 6，可以抓取小论文里的文献了，但是抓取出来不知存在哪里，我看了视频说就在小论文下面，但是没有啊，找了很多地方都没有找到，用起来有点崩溃了，太耗时间了，我想放弃了。建议另外提供一个参考文献和研究成果都可以像论文第一章第二章直接输入那种模板，我感觉那种模板的利用率更高，特别是研究生论文和本科生论文，里面参考文献不多，直接输入也花不了多少时间，也不担心运行出错的问题，上手也很快，多一个版本多一个选择。若是可以，麻烦给我提供一份，非常感谢！

**Reply by lujingping** [reply]
模板非常好用，我遇到的问题作者大神都一对一指导帮解决了，非常感谢！大家可以放心使用，提高效率，省时省力！

**Reply by liuxufeng2016** [reply]
原模版我没有修改，编译生成的PDF中，参考文献是文献标识，而不是数字

**Reply by YanMing-lxb** [reply]
你的编译链是什么

**Reply by YanMing-lxb** [reply]
这个主要是因为你的 bib 文件中没有你引用的参考文献条目，检查一下你的 bib 文件

**Comment by liuxufeng2016** [comment]
我的编译链是
xelatex -&gt; biber -&gt; xelatex*2
谢谢。


刘旭锋
***@***.***



&nbsp;






------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "YanMing-lxb/GUET_Thesis_LaTeX"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年11月22日(星期五) 下午3:10
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [YanMing-lxb/GUET_Thesis_LaTeX] 自己编译参考文献和研究成果的问题 (Discussion #36)





 
你的编译链是什么

**Comment by liuxufeng2016** [comment]
盲审格式要求
删除致谢、学位论文独创性声明、学位论文使用授权声明等页面。
在学期间发表的论文及研究成果只列期刊名称、年份、收录情况、作者排名，不要出现作者或他人姓名，也不能出现论文标题，专利标题，参与项目标题。现有模版没办法做做呢

**Reply by YanMing-lxb** [reply]
注意看文件中的注释说明和README文档

`\documentclass[master, bversion]{GUET-Thesis} ` 其中 `bversion` 是盲审版，你试一下看看
